container/Dockerfile.vllm

```diff
@@ -2,13 +2,11 @@
 # SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-ARG BASE_IMAGE="nvcr.io/nvidia/cuda-dl-base"
-# TODO OPS-612: NCCL will hang with 25.03, so use 25.01 for now
-# Please check https://github.com/ai-dynamo/dynamo/pull/1065
-# for details and reproducer to manually test if the image
-# can be updated to later versions.
-ARG BASE_IMAGE_TAG="25.01-cuda12.8-devel-ubuntu24.04"
-ARG ENABLE_KVBM=false
+# See build.sh's VLLM_BASE_IMAGE and VLLM_BASE_IMAGE_TAG for the actual values.
+ARG BASE_IMAGE
+ARG BASE_IMAGE_TAG
+# See build.sh's ENABLE_KVBM for the actual value.
+ARG ENABLE_KVBM
```

Comment:
I wonder if we would like to default it to `true`, and skip this variable at the `build.sh`, so KVBM installation is solely controlled by the Dockerfile.*?
Not sure if there are scenarios where its installation is not preferred?
cc @ziqifan617